hazelcast/src/main/java/com/hazelcast/map/impl/querycache/subscriber/DefaultQueryCache.java

```diff
@@ -206,7 +205,6 @@ private void destroyRemoteResources() {
     }
 
     private boolean destroyLocalResources() {
-        removeConfig();
```

Comment:
Is it no longer necessary to remove the query cache config from `QueryCacheConfigurator`? The only case in which I see query cache config being removed from `QueryCacheConfigurator` is when an exception is thrown during query cache construction.

Reply:
yes will not remove configs